### DISCUSSION TITLE
Tailwindcss & next/image styling

### DISCUSSION THREAD

**Discussion by wojventures** [vercel/next.js#18467] [Help] [answered]
Hi!

I want to switch all my images to `next/image`, but I don't know what to do about styling. So far I have been styling my images with Tailwind's "w/h" classes, but these are incompatible with new width/height attributes.

So far I'm using `unsized` property, but it doesn't feel right.

What are the best practices for this kind of integration?

**Comment by MathiasKandelborg** [comment, marked as answer]
There's [a PR in the works](https://github.com/vercel/next.js/pull/18491) that will most likely solve your problems.

Look at the changes and see if they're applicable to your use-case. Here are the added layout prop docs at this point (they're obviously subject to change). 

> The rendered layout of the image. If `fixed`, the image dimensions will not change as the viewport changes (no responsiveness). If `intrinsic`, the image will scale the dimensions down for smaller viewports but maintain the original dimensions for larger viewports. If `responsive`, the image will scale the dimensions down for smaller viewports and scale up for larger viewports. Default `intrinsic`.

**Reply by wojventures** [reply]
Thanks, that solves it!
